repository.json is not updated properly

I would like to bring below thing to your notice that, we found an observation in this new v9.6.1 tag which we created is below, this one we didn't observed with v9.4.0, even we face this error we were able to proceed further with v9.6.1

issue in the file: https://github.com/SearchScale/dataimporthandler/commit/e36cd807b61f3ab0c02cafa72b0d1e0a09461b37

When we execute below command 

solr@server1:/opt/solr961/bin$ ./solr package deploy data-import-handler -y -collections test


Executing {"add-requesthandler":{"name":"/dataimport","class":"data-import-handler:org.apache.solr.handler.dataimport.DataImportHandler","defaults":{"config":"data-config.xml"}}} for path:/api/collections/test/config

Executing http://localhost:8983/api/collections/test/config/requestHandler?componentName=/dataimport&meta=true for collection:test
{
  "responseHeader":{
    "status":0,
    "QTime":17},
  "config":{"requestHandler":{"/dataimport":{
        "name":"/dataimport",
        "class":"data-import-handler:org.apache.solr.handler.dataimport.DataImportHandler",
        "defaults":{"config":"data-config.xml"},
        "_packageinfo_":{
          "package":"data-import-handler",
          "version":"9.6.1",
          "files":[
            "/package/data-import-handler/9.6.1/data-import-handler-9.6.1.jar",
            "/package/data-import-handler/9.6.1/mariadb-java-client-2.6.0.jar"],
          "manifest":"/package/data-import-handler/9.6.1/manifest.json",
          "manifestSHA512":"ee8f4fcb1762b36fb7af1b97e76d757f29ad55b3e796db014ba90ce80f283d14e3c56060bd7329b74024615b8b6bef0f75c39c73f6d3e558200e6f6d6daca8be"}}}}}
**com.jayway.jsonpath.InvalidPathException: Error in JSON Path:$[config].[requestHandler].[HANDLER-PATH].[_packageinfo_].[version]**
        at org.apache.solr.packagemanager.PackageManager.jsonPathRead(PackageManager.java:873)
        at org.apache.solr.packagemanager.PackageManager.verify(PackageManager.java:837)
        at org.apache.solr.packagemanager.PackageManager.deployPackage(PackageManager.java:365)
        at org.apache.solr.packagemanager.PackageManager.deploy(PackageManager.java:950)
        at org.apache.solr.cli.PackageTool.runImpl(PackageTool.java:187)
        at org.apache.solr.cli.ToolBase.runTool(ToolBase.java:52)
        at org.apache.solr.cli.SolrCLI.main(SolrCLI.java:154)
**Caused by: com.jayway.jsonpath.InvalidPathException: Could not parse token starting at position 1. Expected ?, ', 0-9, ***
        at com.jayway.jsonpath.internal.path.PathCompiler.fail(PathCompiler.java:642)
        at com.jayway.jsonpath.internal.path.PathCompiler.readNextToken(PathCompiler.java:139)
        at com.jayway.jsonpath.internal.path.PathCompiler.readContextToken(PathCompiler.java:123)
        at com.jayway.jsonpath.internal.path.PathCompiler.compile(PathCompiler.java:58)
        at com.jayway.jsonpath.internal.path.PathCompiler.compile(PathCompiler.java:75)
        at com.jayway.jsonpath.JsonPath.<init>(JsonPath.java:97)
        at com.jayway.jsonpath.JsonPath.compile(JsonPath.java:515)
        at com.jayway.jsonpath.internal.JsonContext.pathFromCache(JsonContext.java:222)
        at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:77)
        at org.apache.solr.packagemanager.PackageManager.jsonPathRead(PackageManager.java:869)
        ... 6 more

**ERROR: Error in JSON Path:$[config].[requestHandler].[HANDLER-PATH].[_packageinfo_].[version]**

**the difference is verify-command  -  condition thing -> in 9.4.0 it is there as RH-HANLDER-PATH but in 9.6.1 i see HANDLER-PATH, also in 9/6/1 we  have in path RH-HANDLER-PATH**
i believe we should have RH-HANDLER-PATH 

**Please correct me if i am wrong.** 


**9.4.0**

  "verify-command": {
                    "condition": "$['config'].['requestHandler'].['${RH-HANDLER-PATH}'].['_packageinfo_'].['version']",
                    "expected": "${package-version}",
                    "method": "GET",
                    "path": "/api/collections/${collection}/config/requestHandler?componentName=${RH-HANDLER-PATH}&meta=true"
                }


**9.6.1**

"verify-command": {
                    "condition": "$[config].[requestHandler].[**HANDLER-PATH**].[_packageinfo_].[version]",
                    "expected": "${package-version}",
                    "method": "GET",
                    "path": "/api/collections/${collection}/config/requestHandler?componentName=${**RH-HANDLER-PATH**}&meta=true"
                }

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -89,7 +89,7 @@ jobs:
                   "verify-command": {
                     "path": "/api/collections/${collection}/config/requestHandler?componentName=${RH-HANDLER-PATH}&meta=true",
                     "method": "GET",
-                    "condition": "$['config'].['requestHandler'].['${RH-HANDLER-PATH}'].['_packageinfo_'].['version']",
+                    "condition": "$['"'"'config'"'"'].['"'"'requestHandler'"'"'].['"'"'${RH-HANDLER-PATH}'"'"'].['"'"'_packageinfo_'"'"'].['"'"'version'"'"']",
                     "expected": "${package-version}"
                   }
                 }

